scripts/ci/travis_build.sh

```diff
@@ -1,19 +1,56 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+
+CURRENT_COMMIT=$(git rev-parse --short HEAD)
+LAST_TAG=$(git describe --abbrev=0 --tags)
+COMMIT_COUNT_SINCE_LAST_TAG=$(git log --oneline ${LAST_TAG}...HEAD | wc -l | bc)
+COMMIT_COUNT_SINCE_v140=$(git log --oneline v1.4.0...HEAD | wc -l | bc)
+
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+source ${DIR}/../version_number.sh
+
 
 if [ -n "${TRAVIS_TAG}" ]; then
   echo "Building release"
   export APP_NAME="QField"
   export PKG_NAME="qfield"
   export APP_ICON="qfield-logo.svg"
+  export APP_VERSION="${TRAVIS_TAG}" #  (v1.2.3 or v1.2.3-rc4)
+
+  # adding the code name to the version string
+  CURRENT_CODENAME=$(cat ${DIR}/../../RELEASE_NAME)
+  APP_VERSION_STR="${TRAVIS_TAG} - ${CURRENT_CODENAME}"
```

Comment:
The initial `v` needs to be stripped from TRAVIS_TAG

Reply:
fixed, use a dedicated and tested function